locations/jclouds/src/main/java/org/apache/brooklyn/core/mgmt/persist/jclouds/JcloudsStoreObjectAccessor.java

```diff
@@ -57,11 +55,13 @@ public boolean exists() {
     @Override
     public void put(String val) {
         if (val==null) val = "";
```

Comment:
In `FileBasedObjectStoreAccessor` you throw an exception on trying to write a null value, should be consistent here.